src/test/java/seedu/address/testutil/TypicalBookings.java

```diff
@@ -10,7 +13,7 @@
     public static final Booking LASTWEEK_YESTERDAY = new BookingBuilder()
         .withBookingPeriod(TypicalBookingPeriods.LASTWEEK_YESTERDAY).build();
 
-    public static final Booking LASTWEEK_YESTERDAY_CHECKED_IN = new BookingBuilder()
+    public static final Booking LASTWEEK_YESTERDAY_CHECKEDIN = new BookingBuilder()
         .withBookingPeriod(TypicalBookingPeriods.LASTWEEK_YESTERDAY)
         .withCheckIn(true).build();
 
```

Comment:
Once you implement `checkout` command with the removal of `Guest`, then these `TypicalBooking`s would have to include different `Guest`s. 

**Note that `Guest` should not repeat across ANY bookings if `checkout` intends to remove the `Guest` without checking through all bookings first. E.g. Amy is booked today-tomorrow, we check her out and delete her. But, she still has a booking for next week.

tldr; Concierge currently does not support simultaneous bookings by the same `Guest`. This is caught by the `DuplicateGuestException` at adding booking.

Reply:
Yes, so `add` command should be refactored to not add the guest to the UniqueGuestList. Are you planning to do this?